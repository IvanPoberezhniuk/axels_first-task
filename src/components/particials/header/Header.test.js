import { unmountComponentAtNode } from 'react-dom';

import { Header } from '../../index';

const setUp = () => render(<Header />);
let component;

beforeEach(() => {
  component = setUp();
});

afterEach(() => {
  unmountComponentAtNode(component);
  component.remove();
  component = null;
});

describe('Header component', () => {
  it('SNAPSHOT', () => {
    expect(component).toMatchSnapshot();
  });
});

