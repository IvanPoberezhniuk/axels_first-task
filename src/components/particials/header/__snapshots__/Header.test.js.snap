// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component SNAPSHOT 1`] = `
<header
  class="sc-gsTCUz jmMyoC"
>
  <h1>
    TEST APP
  </h1>
</header>
`;
